src/mmap.rs

```diff
@@ -212,20 +222,26 @@ impl Bytes<MemoryRegionAddress> for GuestRegionMmap {
     /// * Read bytes from /dev/urandom
     ///
     /// ```
+    /// if cfg!(unix) {
     /// # use vm_memory::{Address, Bytes, GuestAddress, GuestMemoryMmap};
     /// # use std::fs::File;
     /// # use std::path::Path;
+    /// #
     /// # let start_addr = GuestAddress(0x1000);
-    /// # let gm = GuestMemoryMmap::from_ranges(&vec![(start_addr, 0x400)]).unwrap();
-    ///   let mut file = if cfg!(unix) {
-    ///       File::open(Path::new("/dev/urandom")).unwrap()
-    ///   } else {
-    ///       File::open(Path::new("c:\\Windows\\system32\\ntoskrnl.exe")).unwrap()
```

Comment:
Just curious, should we leave the `windows` branch as well? (sorry, I only noticed this change now)

Reply:
Indeed we should, hiding it like the `TempFile` setup. Also copying this over to the other `/dev/urandom` examples.